pulsar-broker/src/main/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesService.java

```diff
@@ -381,8 +422,7 @@ private void initPolicesCache(SystemTopicClient.Reader<PulsarEvent> reader, Comp
                 if (log.isDebugEnabled()) {
                     log.debug("[{}] Reach the end of the system topic.", reader.getSystemTopic().getTopicName());
                 }
-                policyCacheInitMap.computeIfPresent(
-                        reader.getSystemTopic().getTopicName().getNamespaceObject(), (k, v) -> true);
```

Comment:
It doesn't seem to be possible to remove this, only when the execution reaches this line does it mean that init is completed.